evergreen.yml

```diff
@@ -270,18 +286,18 @@ functions:
           echo "------------------------------------"
 
           echo "-- Merging the SBOMs with the licenses information and the SBOM with the  vulnerabilities information in $SBOM_FINAL --"
-          
+
           temp_output="temp_output.json"
           if [[ -f "$temp_output" ]] ; then
               rm "$temp_output"
           fi
           touch $temp_output
-          
+
           while IFS= read -r line
           do
             if [[ "$line" == *"purl"* ]]; then
               bash_purl=$(echo $line | cut -d '"' -f4)
-              command=$(echo "./$SBOM_DIR/jq '.components[] | select(.purl == \"$bash_purl\").licenses' $SBOM_LICENSES")
+              command=$(echo "./$SBOM_DIR/jq '.components[] | select(.purl == \"$bash_purl\").licenses' $SBOM_LICENSES | ./$SBOM_DIR/jq -s 'flatten(1)'")
```

Comment:
I had to update this because ultimately, jq finding something like
```
[ {"license": "MIT"} ] [ {"license": "MIT"} ] 
```
and falling over trying to add the team responsible info because it wasn't comma separated. Adding this additional command combines them into a single array, and allows the write to succeed.

Reply:
Weird. It looks like it was the new version of cargo-cyclonedx which broke the flow.